config/webpack.config.js

```diff
@@ -155,9 +155,7 @@ module.exports = env => {
           unidiff can be removed once https://github.com/mvoss9000/unidiff/pull/1 is merged */
           exclude: /node_modules(?!\/(@novnc|unidiff))/,
           loader: 'babel-loader',
-          options: {
-            presets: [require.resolve('@theforeman/builder/babel')],
-          },
+          options: require.resolve('@theforeman/builder/babel'),
```

Comment:
https://github.com/theforeman/foreman-js/blob/master/packages/builder/babel/index.js#L8 returns `({ presets, plugins })`
which indeed should satisfy the `options` here,

although it worked for two years, so I wonder what changed?